templates/focus-ios.xliff

```diff
@@ -481,6 +558,10 @@
         <source>Open in %@</source>
         <note>Text for the share menu option when a user wants to open a page in Focus.</note>
       </trans-unit>
+      <trans-unit id="ShareMenu.RequestDesktop">
+        <source>Request Desktop Site</source>
+        <note>Text for the share menu option when a user wants to reload the site as a desktop</note>
```

Comment:
Nit: the comment here is a bit awkward. I'm not too sure what "...wants to reload the site as a *desktop* means".